node-project-template/lint-tests.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
-
-focused_tests="$(grep -r -n 'fdescribe(\|fit(' ./test)"
```

Comment:
it also lints other useful stuff (`it()` with no expects, disabled tests, common typos (forgetting parentheses after `expect(foo).toBeUndefined;`)